projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -315,9 +315,16 @@ public static function get_videopress_stats() {
 		}
 
 		$videopress_stats = new VideoPress_Stats();
+		$featured_stats   = $videopress_stats->get_featured_stats( 60 );
```

Comment:
Wondering if we should add a transient cache for this call so we aren't calling it on each page load?

Reply:
Yeah we could do that 🤔 I'll set it to 1 hour? Or would that be too long?